rts/lra/lra-test/src/test/java/io/narayana/lra/participant/SpecIT.java

```diff
@@ -619,7 +619,7 @@ public void testUserData() {
         assertTrue(activity.contains("endData='" + testData));
     }
 
-    @Test
+    //@Test
```

Comment:
would not be better to use `@Ignored` for the maven plugin list the test as existing but ignored?